### PR TITLE
Spell out that running gitea with your own user is unsupported

### PR DESCRIPTION
**Pull request by sztanpet**
I just wanted to run gitea in the easiest way possible, so I ran it with my own user only to find
that I could no longer log in with SSH anymore.

### CODE REVIEW

**Comment by sztanpet** [comment]
For added discoverability for people having the same issue (but not knowing that it is because of Gitea), maybe the error message should be included also, namely `Server refused to allocate pty` (Putty gave me this error, not mentioning Gitea anywhere)?

File: content/page/index.en-us.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ The goal of this project is to make the easiest, fastest, and most painless way
 
 - A cheap Raspberry Pi is powerful enough for basic functionality.
 - 2 CPU cores and 1GB RAM would be the baseline for teamwork.
+- Gitea is supposed to be run with a separate dedicated non-root user account on UNIX systems, no other mode of operation is supported. (**NOTE**: in case you run it with your own user account and the built-in SSH server disabled, Gitea modifies the `~/.ssh/authorized_keys` file, so you will **not** be able to interactively log in.)
```
Comment (tboerger) [inline content/page/index.en-us.md]: separate dedicated? I would prefer just dedicated, than I'm fine with this change.

Comment (sztanpet) [reply content/page/index.en-us.md]: oh totally agreed, plus there is an unneeded comma too

**Comment by bkcsoft** [comment]
LGTM

**Comment by strk** [comment]
It's a pity to not support running from own user, if we can document *all* the issues it's better, or is it only the ssh keys ? (that "regenerate keys" command could be made safer)

**Comment by appleboy** [comment]
@sztanpet Can you also update the other language documents?

**Comment by sztanpet** [comment]
No sorry, I do not know the other languages

**Comment by appleboy** [comment]
@sztanpet OK.

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by lunny** [comment]
Could you send another PR to back port to 1.0?

**Comment by sztanpet** [comment]
there it is

**Comment by tboerger** [comment]
There is an open issue to resolve this issue. Until than we will have this phrase which is warning the users.

**Pull request merged**